examples/simple_example_test/main.cc

```diff
@@ -30,7 +30,7 @@
 
 #define JPG_LEN 44618
 #define INDEX_LEN 10606
-#define INDEX_IMPLICIT_LEN 10817
+#define INDEX_IMPLICIT_LEN 10630
```

Comment:
On another note, it would be nice if the lengths weren't fixed, as windows has crlf endings and that throws off the lengths

Reply:
Yes, I have encountered this problem before.
I forget how it was resolved with the INDEX_LEN macro in windows.

Reply:
Do you know why the `travis-ci` is disabled?

Reply:
> Do you know why the `travis-ci` is disabled?

Nope